modules/api/src/main/java/org/apache/fluo/api/config/FluoConfiguration.java

```diff
@@ -496,23 +575,35 @@ public String getAccumuloJars() {
   }
 
   /**
+   * Set the root for the Hadoop DFS value in property {@value #DFS_ROOT_PROP} 
+   * @param dfsRoot The path for the dfs root eg: hdfs://host:port/path note: may not be empty.
    * @since 1.2.0
    */
   public FluoConfiguration setDfsRoot(String dfsRoot) {
     return setNonEmptyString(DFS_ROOT_PROP, dfsRoot);
   }
 
   /**
+   * Gets the value of property {@value #DFS_ROOT_PROP} if set, otherwise gets
+   * the value of property {@value #DFS_ROOT_DEFAULT}
    * @since 1.2.0
    */
   public String getDfsRoot() {
     return getNonEmptyString(DFS_ROOT_PROP, DFS_ROOT_DEFAULT);
   }
-
+  /**
+   * Set the number of worker threads, must be positive. The default is 10 threads.
```

Comment:
replace `The default is 10 threads.` with `The default is {@value #WORKER_NUM_THREADS_DEFAULT} threads.`